app/code/Magento/Sales/Model/ResourceModel/Status/Collection.php

```diff
@@ -1,6 +1,6 @@
 <?php
 /**
- * Oder statuses grid collection
+ * Order status grid collection
```

Comment:
Please move this description just before the class name.
See \Magento\Sales\Model\ResourceModel\Order\Invoice.php